sklearn/utils/tests/test_multiclass.py

```diff
@@ -590,14 +594,64 @@ def test_ovr_decision_function():
     # assert subset invariance.
     dec_values_one = [
         _ovr_decision_function(
-            np.array([predictions[i]]), np.array([confidences[i]]), n_classes
+            np.array([predictions[i]]), np.array([confidences[i]]), n_classes, []
         )[0]
         for i in range(4)
     ]
 
     assert_allclose(dec_values, dec_values_one, atol=1e-6)
 
 
+def test_ovr_decision_function_confidence_prob():
```

Comment:
I think it could be good to craft some test at the level of the classifier and not only the private function.

Reply:
I understand. I will create a new test for the classifier.